File: Easy-exercises/Skeet Ball/Skeet_Ball.js
const skeetBall = (point, ticket) => {
  let rate;
  rate = ticket * 12;
  rate > point ? console.log("Try again") : console.log("Buy it!");
};
skeetBall(
  parseInt(prompt("Enter number of points :")),
  parseInt(prompt("Enter number of tickets :"))
);

File: Easy-exercises/Skeet Ball/Skeet_Ball.cpp
#include<iostream>
using namespace std;
int main(){
    int point, ticket, rate;
    cout <<"Enter number of points" << endl;
    cin >> point;
    cout <<"Enter number of tickets" <<endl;
    cin >> ticket;
    rate = ticket * 12;
    (rate > point) ? cout <<"Try again" << endl
                   : cout <<"Buy it! " <<endl;
    return 0;
}
